polars/polars-core/src/chunked_array/strings/json_path.rs

```diff
@@ -20,10 +21,29 @@ fn extract_json<'a>(expr: &PathCompiled, json_str: &'a str) -> Option<Cow<'a, st
     })
 }
 
+#[cfg(feature = "extract_jsonpath")]
```

Comment:
Can we move this whole file to the `polars-ops` crate? I want polars-core to only be core algorithms and datatypes. 

So now we are expanding on this functionality it might be the right time to move it immediately.

Reply:
Sounds good. I've moved it to the `polars-ops` crate.